CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Fixes
 
+- `[jest-config]` [**BREAKING**] Add `mts` and `cts` to default `moduleFileExtensions` config ([#14369](https://github.com/facebook/jest/pull/14369))
```

Comment:
To me this feels more like a feature. Nothing is broken, this is just change in the default options.